pravega/zookeeper-operator
# Upgrade to zk 3.8, upgrade to temurin java-11, delete apt cache

Closes #514

### Change log description

Upgrade to zookeeper 3.8. Upgrade to Temurin java-11. Delete apt cache

### Purpose of the change

See #514 

### What the code does

Builds a better docker image

### How to verify it

Build image, verify that it uses newer Zookeeper (3.8), newer Java release (Eclipse Temurin Openjdk-11), newer Linux release (Ubuntu Focal)
